### DISCUSSION TITLE
For Sonny - A png to replace the poor mangle Astronaut in Astroid template manager.

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#496] [General]
@sonvnn 
This file here is a compress png to replace the actual mangle (Check the differences, the left hand is missing part of the glove...) — I remade the Astronaut in Illustrator (I have the vector file if you want it)
The file have the same name and size (1000x1000 pixels) from the original and smaller in weight (Original = 57kb — My replacement = 20kb)
[astronaut.png.zip](https://github.com/templaza/astroid-framework/files/13628077/astronaut.png.zip)

**Good Day to the community!**

![Screen Shot 2023-12-10 at 9 25 38 AM](https://github.com/templaza/astroid-framework/assets/1907276/328cfa73-e685-4be3-8778-40c9ff52866b)

**Comment by sonvnn** [comment]
Thank you @Chacapamac . I updated image in https://github.com/templaza/astroid-framework/commit/248ea78bac0c418b7a59070ea085fb0e6ad98dab

**Reply by Chacapamac** [reply]
Cool!
Good Day!
